core-bundle/src/Resources/contao/dca/tl_page.php

```diff
@@ -544,24 +544,24 @@ static function ($value)
 		(
 			'exclude'                 => true,
 			'inputType'               => 'select',
-			'options'                 => array(''=>'http://', '1'=>'https://'),
+			'options'                 => array(0=>'http://', 1=>'https://'),
```

Comment:
```suggestion
			'options'                 => array('http://', 'https://'),
```

Adding the array keys is redundant, isn‘t it?

Reply:
I'd still prefer it in this case, but I'm fine either way.

Reply:
Ah - actually neither will work. Now that the values are normal integers, the widget will treat the labels as values. I think we'll need a load and save callback here. Or use `reference`.

Reply:
Would `'isAssociative' => true` solve this?

Reply:
@m-vo I'll try that. Otherwise I fixed it in bfe3b49aca5f42bd38e6246dfb9fe6c1860efb06

Reply:
Yes, that works too 👍